docs/internal-documentation/distrib-testing/Ubuntu-nodjs-build-environment.md

```diff
@@ -0,0 +1,29 @@
+# Ubuntu nodjs Build Environment
+
+The requirements to build the Zonemaster-GUI distribution zip file are nodejs
+and npm. Below are instructions to create such a build environment on Ubuntu.
+
+Nodjs and npn are available from the [Node.js] official website. Minimal version
+of Nodejs is 10.0 but install the latest LTS version available. The process has
+been tested on Ubuntu 18.04, which we use here.
+
+1. Make a clean installation of Ubuntu 18.04.
+
+2. Update the package database.
+
+   ```sh
+   sudo apt-get update
+   ```
+
+6. Install nodejs by using [NVM], a node version manager.
+
+   ```sh
+   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+   nvm install 13.12.0
+   nvm use 13.12.0
+   ```
```

Comment:
After running the `curl` command, the `nvm` command was not found. [This is a known issue](https://github.com/nvm-sh/nvm#troubleshooting-on-linux), and reloading the `bashrc` file helped.

Maybe this info should be added to ease the building process :

```
source ~/.bashrc
```

And this can also be added  in the [section 5 of the Release Process document](https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/ReleaseProcess-create-test-distribution.md#5-produce-distribution-zip-file).

Reply:
Fixed.